src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -33,10 +34,40 @@ class AssetsInstallCommand extends ContainerAwareCommand
     const METHOD_ABSOLUTE_SYMLINK = 'absolute symlink';
     const METHOD_RELATIVE_SYMLINK = 'relative symlink';
 
+    private $baseDir;
+    private $filesystem;
+
     /**
-     * @var Filesystem
+     * @param string          $baseDir
+     * @param Filesystem|null $filesystem
      */
-    private $filesystem;
+    public function __construct($baseDir = null, Filesystem $filesystem = null /**, $newApi = false*/)
```

Comment:
Not required to me. Passing the argument to the parent constructor just won't have any effect anymore, it'll just be useless. Php won't cry, only IDEs and SCA would scream a bit, but it's not that much important if you don't notice it (could be mentioned in the UPGRADE file though).